src/models/monitoringReviewStatus.js

```diff
@@ -0,0 +1,42 @@
+const { Model } = require('sequelize');
+
+export default (sequelize, DataTypes) => {
+  class MonitoringReviewStatus extends Model {
+    static associate(models) {
+    }
+  }
+  MonitoringReviewStatus.init({
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    statusId: {
```

Comment:
should this be here? what is the difference between ID and statusId? statusId doesn't seem to link to any other tables...

Reply:
it is the foreign key from the source data to the review: monitoringReview.statusId <=> monitoringReviewStatus.statusId

The ID column is more for us. The column statusId is the main column used in this table.